main.go

```diff
@@ -65,6 +65,7 @@ func main() {
 				SecurityAlerts:        c.Bool("security-alerts"),
 				EventsFile:            os.Stdout,
 				ErrorsFile:            os.Stderr,
+				CaptureStackTraces:    c.Bool("capture-stack-traces"),
```

Comment:
We alreay have a "capture" flag (see 2 lines below).
We can add this as a new capture option - "stack-traces"